.gitignore

```
@@ -0,0 +1 @@
+node_modules
```

We should add a few more files/directories to .gitignore:

```suggestion
node_modules
.DS_Store
package-lock.json
yarn.lock
*.log
test-report.html
```